src/persist-client/src/cli/inspect.rs

```diff
@@ -192,6 +200,14 @@ pub struct StateArgs {
     /// URI scoped to the environment's bucket prefix.
     #[clap(long, env = "BLOB_URI")]
     pub(crate) blob_uri: String,
+
+    /// Inspect the state as of a given sequence number, if possible
+    #[clap(long)]
+    pub(crate) seqno: Option<String>,
+
+    /// Inspect the state rollup with the given ID, if available.
+    #[clap(long)]
+    pub(crate) rollup: Option<String>,
```

Comment:
These naturally don't apply to the other commands that take state args, and there's a bunch. Probably I should split out a new struct... unless you think that those other commands should eventually support these new options, in which case maybe I should just add some assertions that they're currently unused. Thoughts?

Reply:
yeah I think you want `Command::StateRollup(StateRollupArgs)` where `StateRollupArgs` is a new struct that embeds and clap::flattens `StateArgs` instead of it taking `StateArgs` directly